gulp-tasks/sass.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file
+ * This file contains function for sass gulp task
+ *
+ * The gulp task compiles and minifies scss/civicase.scss file into css/civicase.min.css.
+ * Also prefix the output css selector with `#bootstrap-theme` selector except the output.
+ * selector starts from either `body`, `page-civicrm-case` or `.___outside-namespace` classes.
+ */
+
+'use strict';
+
+var autoprefixer = require('gulp-autoprefixer');
+var bulk = require('gulp-sass-bulk-import');
+var civicrmScssRoot = require('civicrm-scssroot')();
+var cssmin = require('gulp-cssmin');
+var gulp = require('gulp');
+var postcss = require('gulp-postcss');
+var postcssDiscardDuplicates = require('postcss-discard-duplicates');
+var postcssPrefix = require('postcss-prefix-selector');
+var rename = require('gulp-rename');
+var sass = require('gulp-sass');
+var stripCssComments = require('gulp-strip-css-comments');
+var sourcemaps = require('gulp-sourcemaps');
+var transformSelectors = require('gulp-transform-selectors');
+
+var bootstrapNamespace = '#bootstrap-theme';
```

Comment:
These two variables are constant, so please have them in upper case
Eg: `BOOTSTRAP_NAMESPACE`